### PR TITLE
Add documentation for $(valorant)

### PR DESCRIPTION
**Pull request by Rydann**

### CODE REVIEW

File: docs/variables/valorant.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+id: valorant
+---
+
+# $(valorant)
+
+Returns a range of metadata around a broadcaster's [**Riot Games**](https://www.riotgames.com/en) account, in relation to the game VALORANT, that can be accessed via multiple different variable members for each unique piece of information.
```
Comment (aidenwallis) [inline docs/variables/valorant.md:7]: * The league one links to the league website, should we add that on `VALORANT`?
* Can you change the riot games link to remove the `en` suffix, I think it should let the users' browser prefs take priority then

File: docs/variables/valorant.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  * `escalation`
+  * `premier`
+  * `replication`
+  * `spikerush`
+  * `swiftplay`
+  * `unrated`
+
+* **Special Modifiers**
+  * `include_custom` *(adds custom games to be considered)*
+  * `custom_only` *(limits the considered games to **custom games only**)*
```
Comment (aidenwallis) [inline docs/variables/valorant.md:27]: The league one includes a `-` between the name and description, should it be formatted the same way?

File: docs/variables/valorant.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+* In case an invalid game mode is specified, returns the following:
+
+    ```
+    [Error: Queue must be one of: competitive, deathmatch, escalation, premier, replication, spikerush, swiftplay, unrated.]
+    ```
+
+## $(valorant.stream_ties)
+
+Returns the amount of games tied that were played in the specified game mode(s) during the current stream.
```
Comment (aidenwallis) [inline docs/variables/valorant.md:115]: Would this read better as

```
Returns the number of games that ended in a tie for a specified game mode(s) during a given stream session.
```

File: docs/variables/valorant.md (hunk 2 above)
Comment (Rydann) [reply docs/variables/valorant.md:27]: sure, lets do it

File: docs/variables/valorant.md (hunk 1 above)
Comment (Rydann) [reply docs/variables/valorant.md:7]: sure, and done

File: docs/variables/valorant.md (hunk 3 above)
Comment (Rydann) [reply docs/variables/valorant.md:115]: sounds good, but I think we should leave it `the current stream session`

**Review by aidenwallis — APPROVED** [review]

**Pull request merged**